bigframes/dataframe.py

```diff
@@ -2666,9 +2666,7 @@ def to_gbq(
                 )
             if_exists = "replace"
 
-            temp_table_ref = bigframes.session._io.bigquery.random_table(
-                self._session._anonymous_dataset
-            )
+            temp_table_ref = self._session._random_table(skip_cleanup=True)
```

Comment:
Let's add a comment here explaining that the user and/or integration tool now owns this temp table we create, so that's why it's not included in cleanup.

Reply:
Added, thanks